www/src/components/account/Roles.tsx

```diff
@@ -160,15 +164,22 @@ export function Roles() {
         {' '}
         <CreateRole q={q} />
       </PageTitle>
-      <BillingLegacyUserBanner feature="Roles" />
-      <List>
-        <Header
-          q={q}
-          setQ={setQ}
+      <BillingLegacyUserBanner feature="roles" />
+      {isAvailable ? (
+        <List>
+          <Header
+            q={q}
+            setQ={setQ}
+          />
+          <RolesInner q={q} />
+        </List>
+      ) : (
+        <BillingFeatureBlockBanner
+          feature="roles"
+          description="Define granular permissions for your organization's users and apply them to groups or individuals."
```

Comment:
Use proper apostrophe
```suggestion
          description="Define granular permissions for your organization’s users and apply them to groups or individuals."
```